tests/python/unittest/test_operator.py

```diff
@@ -36,6 +36,14 @@
 import os
 
 def check_rnn_consistency(cell1, cell2, T, N, I, H, grad_req, rtol=1e-2, atol=1e-4):
+    if default_context().device_type == 'cpu':
+    # NOTE(zixuanweeei): Currently, we don't add `add` requests support on fused mkl-dnn rnn operator.
```

Comment:
Tracking issue?

Reply:
@zixuanweeei

Reply:
We tracked this issue by https://github.com/apache/incubator-mxnet/issues/16578. Some information will be added by another PR. Thanks.